head.lisp

```diff
@@ -56,6 +68,18 @@
                  :test #'frames-overlap-p)
                 do (setf (head-number h) i)
                 collect h)))
+      (and (xlib:query-extension *display* "RANDR")
+             (loop
+                for i = 0 then (1+ i)
```

Comment:
I know the previous loop is written this way but it is simpler to write this clause as `for i from 0`